src/scripts/main.js

```diff
@@ -1,3 +1,58 @@
 'use strict';
 
-// write code here
+function rowToObj(row) {
+  const cells = row.querySelectorAll('td');
+
+  return {
+    name: cells[0]?.textContent.trim() || null,
+    position: cells[1]?.textContent.trim() || null,
+    age: Number(cells[2]?.textContent.trim()) || null,
+    salary: parseFloat(cells[3]?.textContent.replace(/[$,]/g, '')) || null,
+  };
+}
+
+function sortTableArr(a, b, col) {
+  if (a[col] < b[col]) {
```

Comment:
The comparison in `sortTableArr` uses `a[col]` and `b[col]`, assuming `col` is a valid key in the objects. Ensure that `col` corresponds to a valid property name in the objects returned by `rowToObj`. If `col` is derived from the text content of the header, it might not match the object keys directly.